Python/pywarpx/WarpX.py

```diff
@@ -24,6 +24,7 @@
 from .Particles import particles, particles_list
 from ._libwarpx import libwarpx
 
+reduced_diagnostics = Bucket('reduced_diagnostics', _reduced_diagnostics_dict={})
```

Comment:
This should follow the pattern for the other instances and should be defined in a separate file, `ReducedDiagnostics.py`. Use the files `Diagnostics.py` as the starting point.